proto/cache.proto

```diff
@@ -206,6 +206,10 @@ message ScoreCard {
 
     // The type of cache relevant to this result.
     resource.CacheType cache_type = 14;
+
+    // The details of the execution that originally produced this result.
+    build.bazel.remote.execution.v2.ExecutedActionMetadata execution_metadata =
```

Comment:
This data might increase the scorecard entry size by a lot. When storing, will we populate only a subset of the proto?

Reply:
I need the execution started timestamp and the execution completed timestamp. I can store the two timestamps instead to decrease the size.